### ISSUE TITLE
restore main window size/position

### ISSUE THREAD

**Issue by jedie** [michaelestermann/WindowLayoutManager#27]
It seems that the main window size will not restored. Is this not possible?

I have a multi monitor setup and want to spread the IDE on my thee monitors...

**Comment by michaelestermann** [comment]
@jedie Yes, the main window will not be restored. It only stores the tool windows.
But this can be an enhancement for the future. Hence I'll mark this issue with the enhancement tag 👍
